.cirrus.yml

```diff
@@ -999,6 +990,7 @@ upgrade_test_task:
         TEST_FLAVOR: upgrade_test
         DISTRO_NV: ${FEDORA_NAME}
         VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
+        CI_DESIRED_DATABASE: boltdb
```

Comment:
This makes me queasy. Without this, the upgrade test is failing. "No such container", I think. But AIUI this shouldn't be necessary? Will investigate further tomorrow.

Reply:
Because you force sqlite via containers.conf in the CI setup otherwise and the old podman has no idea what sqlite is so it creates containers in boltdb.
The correct thing here would be an empty/no `CI_DESIRED_DATABASE` set.

Reply:
Of course. I really should stop persevering on these things at such late hour. Thank you.